library/src/main/java/com/ricohapi/mstorage/MediaStorage.java

```diff
@@ -256,7 +259,39 @@ public void addMeta(String mediaId, Map<String, String> userMeta, CompletionHand
     }
 
     public void removeMeta(String mediaId, String key, CompletionHandler<Object> handler) {
-        // TODO
+        try {
+            if (accessToken == null) {
+                throw new RicohAPIException(0, "wrong usage: use the connect method to get an access token.");
+            }
+            RicohAPIRequest request;
+            if (key == null) {
```

Comment:
You can also check if mediaId is null. So, it'd be like `mediaId == null || key == null`. 
Maybe we should check mediaId in other methods.